aea/agent_loop.py

```diff
@@ -80,7 +74,6 @@ def start(self) -> None:
     async def run_loop(self) -> None:
         """Run agent loop."""
         self.logger.debug("agent loop started")
-        self._state.set(AgentLoopStates.started)
```

Comment:
why removed?

Reply:
not removed, but moved to another place, to the read inbox messages. 
it reflects readiness better